pkg/provider/example/cmd/BUILD.bazel

```diff
@@ -0,0 +1,27 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
Any particular reason for embedding the `cmd` directory under `pkg/provider/example` rather than an `example` subdirectory under the top level `cmd` directory?

Reply:
moved to cmd folder, thx